website/widgets/form/textbox.md

```diff
@@ -137,3 +137,28 @@ q.page['example'] = ui.form_card(
                       value="I have spellcheck disabld", spellcheck=False)]
 )
 ```
+
+## Mobile keyboard layout
+
+Show proper keyboard layout on mobile devices with `keyboard` attribute. Defaults to `text`.
+This does not prevent user from typing any character. If you want to allow typing e.g. numeric characters only, use in combination with [mask](#with-mask).
```

Comment:
Why mask and not spinbox?

Reply:
Spinbox  currently does not support numeric keyboard layout on iOS. 
Also depends on what kind of number you need - for credit card numbers can be ui.textbox with mask more suitable.
But you are right, spinbox is more suitable for numeric only values. Better deleting the last sentence.